acousticbrainz/testing.py

```diff
@@ -21,16 +22,17 @@ def tearDown(self):
         self.truncate_all()
 
     def truncate_all(self):
-        conn = psycopg2.connect(self.app.config["PG_CONNECT"])
-        cur = conn.cursor()
-        cur.execute("TRUNCATE highlevel_json CASCADE;")
-        cur.execute("TRUNCATE highlevel CASCADE;")
-        cur.execute("TRUNCATE lowlevel CASCADE;")
-        cur.execute("TRUNCATE statistics CASCADE;")
-        cur.execute("TRUNCATE incremental_dumps CASCADE;")
-        conn.commit()
-        cur.close()
-        conn.close()
+        connection = psycopg2.connect(current_app.config['PG_CONNECT'])
```

Comment:
Ugh. I think I would prefer if it connected to PG_CONNECT_TEST, rather than rebinding PG_CONNECT. If for some reason that rebind doesn't happen, a non-test (production!) DB could get nuked. :(